lxd/patches.go

```diff
@@ -325,7 +325,8 @@ func patchNetworkACLRemoveDefaults(name string, d *Daemon) error {
 
 				// Write back modified config if needed.
 				if modified {
-					err = tx.UpdateNetworkACL(ctx, aclID, &acl.NetworkACLPut)
+					aclPut := acl.Writable()
+					err = tx.UpdateNetworkACL(ctx, aclID, &aclPut)
```

Comment:
Is there any reason *UpdateX* functions accept pointers to Put structures? 
Is it possible that some of them modify contents of the provided structure? Because in such case the updated Put struct has to be written back to the Get struct.

Reply:
Yeah i think it would be clearer to not accept a pointer where these functions dont modify the struct.